# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../qtparted/no-unsermake-detect-autoconf.patch
# Copyright (C) 2006 - 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./admin/detect-autoconf.sh.orig	2005-07-06 19:49:51.000000000 +0300
+++ ./admin/detect-autoconf.sh	2006-12-15 04:50:58.748510750 +0200
#Patch from Gentoo josejx 
@@ -10,8 +10,8 @@
 # Please add higher versions first. The last version number is the minimum
 # needed to compile KDE. Do not forget to include the name/version #
 # separator if one is present, e.g. -1.2 where - is the separator.
-KDE_AUTOCONF_VERS="-2.58 -2.57 257 -2.54 -2.53a -2.53 -2.52 -2.5x"
-KDE_AUTOMAKE_VERS="-1.7 17 -1.6"
+KDE_AUTOCONF_VERS="-2.60 -2.58 -2.57 257 -2.54 -2.53a -2.53 -2.52 -2.5x"
+KDE_AUTOMAKE_VERS="-1.9 -1.7 17 -1.6"
 
 # We don't use variable here for remembering the type ... strings. Local 
 # variables are not that portable, but we fear namespace issues with our
